### PR TITLE
d2l-meter-circle

### PR DESCRIPTION
**Pull request by m6jones**
image

Out of scope:
Animation
RTL
x/y or % choice

### CODE REVIEW

File: components/meter/meter-circle.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		:host {
+			display: inline-block;
+			width: 2.4rem;
+		}
+		.d2l-meter-radial-full-bar,
+		.d2l-meter-radial-progress-bar {
+			fill: none;
+			stroke-linecap: round;
+			stroke-width: 6;
+		}
```
Comment (dlockhart) [inline components/meter/meter-circle.js]: These should probably be `-circle-` to match the component name.

File: components/meter/meter-circle.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		.d2l-meter-radial-text {
+			fill: var(--d2l-color-ferrite);
+			font-size: 0.6rem;
+			line-height: 0;
+		}
+	` ];
+	}
+
+	render() {
+		const lengthOfLine = 21 * 3.14 * 2; // approximation perimeter of circle
```
Comment (dlockhart) [inline components/meter/meter-circle.js]: If you want something more precise: [Math.PI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI)

Comment (m6jones) [reply components/meter/meter-circle.js]: I don't think we need the precision but for readability this is probably way better.

**Comment by dbatiste** [comment]
A little tip: if you're ever looking to get element screenshots that are sized perfectly for an element, Chrome Dev Tools supports `Cmd+Shift+P` - `Capture node screenshot`.

**Comment by m6jones** [comment]
> A little tip: if you're ever looking to get element screenshots that are sized perfectly for an element, Chrome Dev Tools supports `Cmd+Shift+P` - `Capture node screenshot`.

For the readme I've been using the generated goldens. Maybe this can be automated to load new screenshots.

**Pull request merged**